hvplot/tests/testui.py

```diff
@@ -269,6 +274,40 @@ def test_explorer_code_gridded_dataarray():
         ```"""
     )
 
+def test_explorer_code_opts():
+    ds = xr.tutorial.open_dataset("air_temperature")["air"]
+    explorer = hvplot.explorer(ds, x="lon", y="lat", kind="image", opts={"color_levels": 3})
+    explorer._code()
+    code = explorer.code
+    assert code == dedent("""\
+        da.hvplot(
+            colorbar=True,
+            groupby=['time'],
+            kind='image',
+            opts={'color_levels': 3},
+            x='lon',
+            y='lat'
+        ).opts(
+            color_levels=3
+        )""")
+
+    assert explorer._code_pane.object == dedent("""\
+        ```python
+        import hvplot.xarray
+
+        da.hvplot(
```

Comment:
Why are the options present twice, in the `hvplot(...)` call and in `hvplot().opts(...)`?

Reply:
Unintended; fixed now.